src/styles.scss

```diff
@@ -1 +1,19 @@
-@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+@import "./variables.scss";
+
+@import "../src/main/main.scss";
```

Comment:
If those are your main styles, move them to `src/index.scss` and don't import them anywhere else than in `index.js`.